custom_widgets/annotation_manager/annotationManager.py

```diff
@@ -31,33 +29,49 @@ def __init__(self, app, ui, themePaletteColours: dict, fontRegular, fontTitle):
         self.fontTitle = fontTitle
 
         self._classes = []
-        self.classListItems = []
+        self.classSelected = ""
+        self.annotationSelected = ""
 
         # Setup stylesheet of frame
         self.__setupStyleSheet()
- 
+
         # Connect signals and slots
         self.classSearchLineEdit.textChanged.connect(lambda searchInput: self.__searchForClass(searchInput))
-        self.ui.editPageBtn.toggled.connect(lambda toggled: self.classSelectionListWidget.setEditMode(toggled))
 
     def reset(self) -> None:
-        """ Resets the widget to an empty state """
-        self.classSelectionListWidget.clear()
+        """ Clears all class and annotation items from the tree widget. """
+        self.classTreeWidget.clear()
         self._classes = []
-        self.classListItems = []
 
     def generateClassItems(self) -> None:
         """ Generates the class items for the widget """
         for _class in self.app.project.classesDataset:
             if _class.className not in self._classes:
                 self.generateClassItem(_class.className, _class.classColour)
 
-    def generateClassItem(self, className: str, classColour: tuple ) -> None:
+    def generateClassItem(self, className: str, classColour: tuple) -> None:
         """ Generates a class item """
         if className not in self._classes:
-            self.addClassSelectionListItem(className, classColour)
+            self.addClass(className, classColour)
             self._classes.append(className)
 
+    def addClass(self, className: str, classColour: tuple) -> None:
```

Comment:
```suggestion
    def addClassItem(self, className: str, classColour: tuple) -> None:
```
Just to be explicit that we are not adding a class to the project but a class item - only because there is add class functionality in this widget